### ISSUE TITLE
Passenger indicator on "Ship Info" screen unclear/wrong position.

### ISSUE THREAD

**Issue by adegans** [endless-sky/endless-sky#7868]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Yesterday I was play testing my plugin and couldn't find any indication on how much passengers I'm carrying.
It's there, just in a stupid spot - Which is the "bug".

### Steps to Reproduce

Look at the ship info (press I, ship info tab) 

You see a bunch of the ship specs on the left. Fittings in the middle.
And the top right has which and how many cargo you have. But where is the passegner indicator..?

Turns out it's hidden in the lower right... Took me 30 minutes throughout various missions to figure that out.

### Expected Behavior

For passengers to have their own little heading below "Cargo" on the right side of the window. (See screenshots)
This way it's easy to find and read. No need to separate different kinds of passenger (farmer, miner, family or whatever) I don't think. Just make the placement more clear.

Lumping them in with cargo as a cargo type would work as well. But they're not really cargo in that sense and have their own hold/space (don't take up cargo space)... So that doesn't seem right.

### Screenshots

Change:
<img width="1792" alt="Screenshot 2022-12-12 at 17 54 50" src="https://user-images.githubusercontent.com/9438487/207199765-9e90f7f1-2fb6-4753-b575-fe4b67353304.png">

to this:
<img width="1792" alt="Screenshot 2022-12-12 at 17 54 50" src="https://user-images.githubusercontent.com/9438487/207200115-576b34d0-0f4d-493a-b524-c8873849b840.png">

### Operating System

macOS

### Game Source

GitHub Releases

### Game Version

9.16.1

**Comment by adegans** [comment]
Assuming you're talking about some mod you've made.
I kinda like the look of it though.

To save space, mission cargo can just be combined on 1 line for the cargohold.
That saves a header, empty space/padding etc.
And Passengers split out by planet is nice, but potentially takes up a lot of space as well. So combining that would save space.

**Comment by vitalchip** [comment]
Yes, you can save space that way for sure but there's always the possibility that *something* on the panel will break the 'Y' limits, most likely the ship stats.  

If the panel scrolled then everything gets the benefit of more or less unlinited 'Y' spce and that adds the bonus of being able to format things nicely.

FWIW I've never run out of space for cargo/mission cargo passengers etc, it's always ship stats.

**Comment by johndh** [comment]
Related downstream issues: 
https://github.com/thewierdnut/endless-mobile/issues/14
https://github.com/thewierdnut/endless-mobile/issues/17

Making the interface more scalable and scrollable would be very helpful for smaller screens. I always find myself reducing my fleet size just so that I can keep track of them on the info screen.
